WooCommerce/Classes/ViewRelated/Dashboard/Settings/CardReaders/CardReaderSettingsViewModel.swift

```diff
@@ -123,7 +123,7 @@ final class CardReaderSettingsViewModel: ObservableObject {
 
         // TODO - show the multiple-readers-found UITableView when more than one reader is found
         // For now, just show the tail (last) of the array
-        guard let cardReader = cardReaders.last else {
+        guard let cardReader = cardReaders.first else {
```

Comment:
I'm curious - no need to hold up the commit for this - but I assume this doesn't matter when a single reader is present? (Will try later myself)

Reply:
Good point. it does not. It is one of the things that I tried when tinkering with the code trying to figure out why I was not able to create an intent using the SDK test mode. It probably shouldn't have made it into the diff.

And yes, it should not make a difference when there is only one reader.